plugins/katello/3.10/user_guide/remote_databases/index.md

```diff
@@ -169,7 +169,7 @@ Follow the instructions to [prepare remote mongo](plugins/katello/{{ page.versio
 ### Stop the Foreman server
 Stop the Foreman related services to minimize risk of the data changes during the migration
 ```
-katello-service stop
```

Comment:
I wouldn't touch such an old release altogether. Probably best to keep it to supported releases (3.11 and higher).

Reply:
+1

Reply:
Oh yes. Sorry. This happened by mistake. me--

Reply:
Oh so foreman maintain is supported from 3.11 and not from 3.7?

Reply:
I don't know to be honest, but 3.10 and older are no longer supported so we usually don't update the documentation either.